runner/runner.go

```diff
@@ -148,9 +148,10 @@ func updateScheduler(test string, duration float64) {
 
 func getSchedule(tests []string) ([]string, error) {
 	var (
+		userName    = os.Getenv("CIRCLE_PROJECT_USERNAME")
 		project     = os.Getenv("CIRCLE_PROJECT_REPONAME")
 		buildNum    = os.Getenv("CIRCLE_BUILD_NUM")
-		testRun     = project + "-integration-" + buildNum
+		testRun     = userName + "-" + project + "-integration-" + buildNum
```

Comment:
The CircleCI "buildNum" is not unique globally but within a repository.

So a "testRun" name can theoretically collide between a build from https://github.com/weaveworks/scope/ and another from https://github.com/kinvolk/scope/ or others.

And the runner hardcodes the same scheduler at positive-cocoa-90213.appspot.com to schedule jobs. AFAIU, the scheduler has a database remembering the previous tests and it might be confused when asked to schedule a job already finished months ago. I have not seen the problem in practice yet, but I remember a previous similar issue on https://github.com/weaveworks/scope/pull/1655#issuecomment-231165015